packages/material-ui/src/Popover/Popover.test.js

```diff
@@ -465,7 +457,7 @@ describe('<Popover />', () => {
       openPopover = anchorOrigin =>
         new Promise(resolve => {
           wrapper = mount(
-            <Popover
+            <Popover.Original
```

Comment:
Where is this coming from? The purpose of tests is to test usage.

Reply:
This is bypassing the fake withStyles implementation, we need to inject the CSS into the page, the test fails otherwise on layout dimension assertions.